app/components/Detail.tsx

```diff
@@ -1,13 +1,7 @@
-import { Text } from "@mantine/core";
-
 export function Detail({ children }: { children: React.ReactNode }) {
   return <div className="flex flex-col">{children}</div>;
 }
 
 Detail.Title = function DetailTitle({ children }: { children: React.ReactNode }) {
-  return (
-    <Text size="xs" color="gray" weight={500} transform="uppercase" className="mb-2">
-      {children}
-    </Text>
-  );
+  return <p className="mb-2 text-xs text-[#666666] uppercase">{children}</p>;
```

Comment:
Let's use the tailwind presets for colors as much as possible, `text-neutral-700`, instead of custom hex codes.